doc/source/index.rst

```diff
@@ -187,8 +187,188 @@ the default Docker image to ``rapidsai/rapidsai:latest``, if you're using a cust
 you must ensure the NVIDIA CUDA toolkit is installed with a version that matches the host machine
 along with ``dask-cuda``.
 
+
+Azure
+-----
+
+In order to start using ``dask_cloudprovider.AzureMLCluster`` you need, at a minimum,
+an `Azure subscription <https://azure.microsoft.com/en-us/free/services/machine-learning/>`_,
+an `AzureML workspace <https://docs.microsoft.com/en-us/python/api/azureml-core/azureml.core.workspace.workspace?view=azure-ml-py>`_, and
+a `quota <https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/azure-subscription-service-limits>`_ to create your compute target.
+
+Connecting via SSH key pair does not work on Windows Subsystem for Linux (WSL). It is recommended to use a Compute Instance <https://aka.ms/aml/computeinstance> on the same Azure Virtual Network as the Dask cluster.
+
+Getting started
+^^^^^^^^^^^^^^^
+
+Imports
+~~~~~~~
+
+First, import all necessary modules.
+
+.. code-block:: python
+
+   from azureml.core import Workspace, Environment
+   from azureml.core.conda_dependencies import CondaDependencies
+   from azureml.core.compute import ComputeTarget, AmlCompute
+   
+   from dask_cloudprovider import AzureMLCluster
+
+   import os
+
+Setup
+~~~~~
+
+Next, create the ``Workspace`` object given your AzureML ``Workspace`` parameters. Check
```

Comment:
Could we add some instructions here on how to find this info.

For example I grabbed the `config.json` file from the Azure Portal.

![image](https://user-images.githubusercontent.com/1610850/85581864-df5a1980-b634-11ea-914f-b7392b04c4ed.png)